### PR TITLE
Login area min width prevents resizing from errors

### PR DESCRIPTION
**Pull request by gregbenner**
Login area min width prevents resizing from errors.

TP: https://rockabox.tpondemand.com/entity/11742

### CODE REVIEW

File: src/rb-login-form/rb-login-form.css [hunk 1]
```diff
@@ -12,6 +12,11 @@
  * 1. Comment
  */
 
+/* Hack CSS */
+.hack-width {
```
Comment (jonman) [inline src/rb-login-form/rb-login-form.css]: This is a permanent fix rather than a hack? I assume that the width will not change in the future.

Comment (gregbenner) [inline src/rb-login-form/rb-login-form.css]: @jonman  I think @ohskylab might have a better solution, The width selected is for the current longest error message. I can refactor as a more permanent solution if it's acceptable?

Comment (jonman) [inline src/rb-login-form/rb-login-form.css]: Oh I see. In that case could you make it confirm to his class name convention. `.hackWidth`?

**Comment by gregbenner** [comment]
@jonman cool cool, updated className's

**Pull request merged**